### ISSUE TITLE
Classify ad interests

### ISSUE THREAD

**Issue by kaustubhhiware** [kaustubhhiware/facebook-archive#10]
Classify a facebook user's ad interests into categories, say sports, engineering, humor, developing, etc.

This issue has a lot of potential, and is currently only in ideation phase. Please discuss your vision about how / what can be done here.

**Comment by riya2908** [comment]
For classifying ad interests , we can use collaborative filtering here (specifically item based one)  since it is basically a recommendation system. We can use scikit for finding similarities.Like if there is a user A and he/she searches or likes posts related to humor then we can create a system where he/she will receive the suggestions based on the tags on the other posts like humor can have tags like comedy , funny , fun etc..

**Comment by xprilion** [comment]
@riya2908 I agree with you. Have you downloaded your fb data in json format and checked the ads interest files? can you chalk about briefly which and how will you be using those files? We'll then assign you the task :)

**Comment by riya2908** [comment]
@xprilion I haven't downloaded my dataset , but we can gather the dataset of any user or find it online.I am not active on much social platforms.
However , I do have some path in my head.I'll jot down the pointers and will send them in slack today.

**Comment by xprilion** [comment]
@riya2908 I suggest you to download your facebook data archive in json format before planning for the solution to this issue. It will help you to develop a solution in sync with the project. do ping in the slack channel for any queries or help you might need.

**Comment by riya2908** [comment]
@xprilion yes sure.Thanks
